tests/unit_tests/udf_wrapper_params/filling_mask/multiple_model_single_batch_complete.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import PurePosixPath
+from exasol_udf_mock_python.connection import Connection
+from tests.unit_tests.udf_wrapper_params.filling_mask.mock_filling_mask import \
+    MockFillingMaskFactory, MockFillingMaskModel, MockPipeline
+
+
+def udf_wrapper():
+    from exasol_udf_mock_python.udf_context import UDFContext
+    from exasol_transformers_extension.udfs.models.filling_mask_udf import \
+        FillingMask
+    from tests.unit_tests.udf_wrapper_params.filling_mask.mock_sequence_tokenizer \
+        import MockSequenceTokenizer
+    from tests.unit_tests.udf_wrapper_params.filling_mask.\
+        multiple_model_single_batch_complete import \
+        MultipleModelSingleBatchComplete as params
+
+    udf = FillingMask(
+        exa,
+        batch_size=params.batch_size,
+        pipeline=params.mock_pipeline,
+        base_model=params.mock_factory,
+        tokenizer=MockSequenceTokenizer)
+
+    def run(ctx: UDFContext):
+        udf.run(ctx)
+
+
+class MultipleModelSingleBatchComplete:
+    """
+    multiple model, single batch, last batch complete
+    """
+    batch_size = 4
+    data_size = 2
+    top_k1 = 3
```

Comment:
You have dedicated took tests, so I would use here the same k, such that they differ